### PR TITLE
Displayed Univalent Graphs

### PR DESCRIPTION
**Pull request by Schippmunk**
This pr contains Evan's version of DUARels from #463 plus additional theorems from my thesis.

### CODE REVIEW

**Comment by ecavallo** [comment]
Nice! To satisfy Travis you should add `import Cubical.Displayed.Everything` to `Cubical/README.agda` (but don't create a `Cubical/Displayed/Everything.agda` file).

File: Cubical/Foundations/Transport.agda [hunk 1]
```diff
@@ -67,9 +67,13 @@ isEquivTransport {A = A} {B = B} p =
 transportEquiv : ∀ {ℓ} {A B : Type ℓ} → A ≡ B → A ≃ B
 transportEquiv p = (transport p , isEquivTransport p)
 
-substEquiv : ∀ {ℓ ℓ'} {A B : Type ℓ} (P : Type ℓ → Type ℓ') (p : A ≡ B) → P A ≃ P B
+
+substEquiv : ∀ {ℓ ℓ'} {A : Type ℓ} {a a' : A} (P : A → Type ℓ') (p : a ≡ a') → P a ≃ P a'
 substEquiv P p = (subst P p , isEquivTransport (λ i → P (p i)))
 
+substEquiv2 : ∀ {ℓ ℓ'} {A B : Type ℓ} (P : Type ℓ → Type ℓ') (p : A ≡ B) → P A ≃ P B
```
Comment (ecavallo) [inline Cubical/Foundations/Transport.agda:74]: This `substEquiv2` isn't used anywhere and is subsumed by `substEquiv`, could you remove it?

**Review by ecavallo — CHANGES_REQUESTED** [review]
Looks great! Just one comment to address and then I'll merge.

File: Cubical/Foundations/Transport.agda (hunk 1 above)
Comment (Schippmunk) [reply Cubical/Foundations/Transport.agda]: Sure!

**Review by ecavallo — APPROVED** [review]

**Pull request merged**